src/sdb.h

```diff
@@ -32,8 +32,10 @@ extern "C" {
 #include <sys/types.h>
 #include <sys/stat.h>
 #include <io.h>
-extern void *__cdecl _aligned_malloc(size_t, size_t);
+extern __attribute__((dllimport)) void *__cdecl _aligned_malloc(size_t, size_t);
 extern char *strdup (const char *);
+#undef r_offsetof
+#define r_offsetof(type, member) ((unsigned long) (ut64)&((type*)0)->member)
```

Comment:
i dont understand why you wanna cast a pointer to a 64bit number and then wrap it to 32bit to just get the delta between a type and the field? also, maybe im blind but im grepping for offsetof and i see 0 results